themes/ansible-community/sass/_homepage-blog-band.scss

```diff
@@ -24,3 +24,8 @@
     text-align: left;
   }
 }
+
+.blog-author {
+  font-size: 1.25rem;
```

Comment:
This seems kind of big to me. I'd prefer `1rem` or maybe `0.9`

Reply:
Sure, I put it in a class rather than hardcoding for h4-ish so we could tweak it. Do you want me to add that here, or leave it for the general styling pass?

Reply:
I added it while fixing the other point. Not sure I like how much it looks appears to be part of the excerpt below it, but we can fix that in the styling pass, probably just needs a bit of spacing.